resolwe_bio/processes/import_data/basespace.yml

```diff
@@ -0,0 +1,50 @@
+- slug: basespace-import
+  name: BaseSpace Files Import
+  requirements:
+    expression-engine: jinja
+    executor:
+      docker:
+        image: resolwebio/utils
+    resources:
+      network: true
+      secrets: true
+  data_name: 'BaseSpace Import ({{file_ids|length}} {% if file_ids|length == 1 %}file{% else %}files{% endif %})'
+  version: 0.0.1
+  type: data:basespace
+  category: upload
+  persistence: RAW
+  description: >
+    Import files from BaseSpace.
+  input:
+    - name: file_ids
+      label: BaseSpace file IDs
+      type: list:basic:string
```

Comment:
If file IDs are always integers the input type should be set accordingly.

Reply:
BaseSpace serves file IDs as [strings](https://developer.basespace.illumina.com/docs/content/documentation/rest-api/v1-api-reference#GETfilesId), not integers, so I'd rather not change type.